en/2/00-overview.md

```diff
@@ -10,5 +10,5 @@ Impressive, human! You're a better coder than I thought.
 
```

Comment:
We still have the wrapping up thing. At the end of the lesson we should have this version of the contract deployed so that the user can test things out, right?

Reply:
in a later version when the rust interactor is finished